### PR TITLE
Added symfony 7 and php 8.3. Annotations changed to attribute

### PR DESCRIPTION
**Pull request by slc1**
Changed annotations to attribute. 
Removed sensio/framework-extra-bundle
Made compatibility with php 8.2

### CODE REVIEW

**Comment by mcfedr** [comment]
I think need to adjust the tests to run with correct versions

File: src/Mcfedr/AwsPushBundle/Command/RemoveDisabledCommand.php [hunk 1]
```diff
@@ -6,21 +6,21 @@
 
 use Aws\Sns\SnsClient;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+#[AsCommand('kidslox:account:mcfedr:aws:remove', 'Remove disabled devices')]
```
Comment (deguif) [inline src/Mcfedr/AwsPushBundle/Command/RemoveDisabledCommand.php:14]: The command name has changed with this commit. Is it expected?

File: src/Mcfedr/AwsPushBundle/Command/SubscribeTopicsCommand.php [hunk 2]
```diff
@@ -7,15 +7,15 @@
 use Aws\Sns\Exception\SnsException;
 use Aws\Sns\SnsClient;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
+#[AsCommand('kidslox:account:mcfedr:aws:subscribe', 'Subscribe existing devices to the topic')]
```
Comment (deguif) [inline src/Mcfedr/AwsPushBundle/Command/SubscribeTopicsCommand.php:16]: The command name has changed with this commit. Is it expected?

File: src/Mcfedr/AwsPushBundle/Command/EnableAllCommand.php [hunk 3]
```diff
@@ -6,21 +6,21 @@
 
 use Aws\Sns\SnsClient;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+#[AsCommand('kidslox:account:mcfedr:aws:enable', 'Reenable all devices')]
```
Comment (deguif) [inline src/Mcfedr/AwsPushBundle/Command/EnableAllCommand.php:14]: The command name has changed with this commit. Is it expected?

Comment (mcfedr) [reply src/Mcfedr/AwsPushBundle/Command/EnableAllCommand.php:14]: @slc1 its weird the command names where changed. I think that needs reverting

Comment (deguif) [reply src/Mcfedr/AwsPushBundle/Command/EnableAllCommand.php:14]: I can make a PR to fix that

Comment (deguif) [reply src/Mcfedr/AwsPushBundle/Command/EnableAllCommand.php:14]: Here we go: #48

**Pull request merged**